core/sql/arkcmp/CmpStoredProc.cpp

```diff
@@ -733,7 +733,7 @@ CmpStoredProc::ExecStatus CmpStoredProc::close(CmpSPDataObject&)
 // Methods of CmpISPFuncs
 // -----------------------------------------------------------------------
 
-NAList<CmpISPFuncs::ProcFuncsStruct> CmpISPFuncs::procFuncsArray_(256);
+NAList<CmpISPFuncs::ProcFuncsStruct> CmpISPFuncs::procFuncsArray_(NULL,256);
```

Comment:
I tried this change. Had to include Globals.h in this file. Build went fine, but RMS would not come up on sqstart. Did not investigate further. Did try clean build a couple of times, so I am reasonably sure it is not an env issue.